packages/eslint-plugin-jest/src/index.js

```diff
@@ -8,11 +8,33 @@
  * @flow
  */
 
-/* eslint-disable global-require */
-
 'use strict';
 
-module.exports.rules = {
-  'no-exclusive-tests': require('./rules/no-exclusive-tests'),
-  'no-identical-title': require('./rules/no-identical-title'),
+module.exports = {
+  environments: {
```

Comment:
I'll eventually post an issue about this. @vjeux had some ideas as well, we should do it :)